### DISCUSSION TITLE
v1.98.7

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#518] [Show and tell]
# [AMD]
* [Family 1Ah][Granite Ridge] 
  * P-State programming fix
  * Merge PCI identifier lists
* Reserve the `BTC-NOBR` aggregation to Zen2 architecture

# [AArch64]
* Query and JSON export Hypervisor Configuration Register `HCR_EL2`
  * Experimental mode required

----

# _CoreFreq_ ISO
Instructions in Wiki / [LiveCD](https://github.com/cyring/CoreFreq/wiki/Live-CD)
SHA1 of the attached image
```
dd583e8855f98d9c62dba1b25a099c5f22dd2738
```

<hr /><em>This discussion was created from the release <a href='https://github.com/cyring/CoreFreq/releases/tag/1.98.7'>v1.98.7</a>.</em>

**Comment by cyring** [comment]
@bubbleguuum @inglor @ich777

This version contains a necessary P-State fix for Zen5
So far only tested with Ryzen 7 9800X3D #517

**Reply by ich777** [reply]
@cyring automatic build process just kicked in. ;)

**Reply by cyring** [reply]
Thank you very much

**Comment by bubbleguuum** [comment]
Updated :)

**Reply by cyring** [reply]
Thanks

**Comment by huajian628** [comment]
great

**Comment by huajian628** [comment]
![企业微信截图_17326971932611](https://github.com/user-attachments/assets/20b366dd-7e6a-4b98-b568-e4565fe2a6a1)

**Reply by cyring** [reply]
Great screenshot; thank you

We clearly see on this processor that we're getting temperature per eight CPUs cluster.

**Comment by huajian628** [comment]
![Uploading 企业微信截图_17326969085798.png…]()
